apps/storybook-react/stories/WalletHome.stories.tsx

```diff
@@ -46,27 +51,39 @@ type Story = StoryObj;
 
 const WalletHome: React.FC = () => {
   return (
-    <div className="min-h-screen md:flex md:items-center md:justify-center md:bg-alternative md:py-4">
+    <Box className="min-h-screen md:flex md:items-center md:justify-center md:bg-alternative md:py-4">
       {/* Container Expanded View */}
-      <div className="mx-auto w-full bg-default md:max-w-xl md:rounded-3xl md:py-4">
+      <Box className="mx-auto w-full bg-default md:max-w-xl md:rounded-3xl md:py-4">
         {/* Header */}
-        <div className="border-color-border-muted px-4 py-4 md:px-8">
-          <div className="flex items-center justify-between">
-            <div className="flex items-center space-x-2 overflow-hidden">
+        <Box className="border-color-border-muted px-4 py-4 md:px-8">
```

Comment:
Notice this Box doesn't use flex so it was breaking the layout of WalletHome

<img width="1512" alt="Screenshot 2025-06-06 at 1 08 07 PM" src="https://github.com/user-attachments/assets/65a8f45d-6603-4e60-828e-f880b068e1c1" />

<img width="1501" alt="Screenshot 2025-06-06 at 1 08 46 PM" src="https://github.com/user-attachments/assets/f393f2fa-7ac3-4245-8608-fbcd882c6e1c" />